### PR TITLE
Added a welcome screen

### LINKED ISSUE #38: A welcome screen
just like the official pomodoro app a screen must appear on app startup with text somewhat like "Welcome to Tametu"
or with text which you like to inculcate.

###Would you like to work on this issue?
yes

### PR DESCRIPTION
**Pull request by chanmol1999**
## Added a welcome screen of 1 second timeout.

Fixes #38 

## Type of change
Just put an x in the [] which are valid.
- [ x] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -6,47 +6,50 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
 
     <application
-        android:name="gis2018.udacity.tametu.App"
+        android:name=".App"
```
Comment (numerative) [inline app/src/main/AndroidManifest.xml]: Why this change?

File: app/src/main/res/layout/activity_first.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        android:layout_height="62dp"
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="435dp"
+        android:fontFamily="@font/quicksand_medium"
+        android:gravity="center"
+        android:text="Tametu Timer"
+        android:textColor="#FFFFFF"
+        android:textSize="32sp" />
+
```
Comment (numerative) [inline app/src/main/res/layout/activity_first.xml:59]: Clean code: why so empty?

**Review by numerative — COMMENTED** [review]
Can you please look into these?

**Comment by chanmol1999** [comment]
please review it now .It was my negligence.

**Comment by numerative** [comment]
Merged after making a minor change:
- increased the timeout to: 1200ms :sweat_smile: 

Closing this manually.
